app/src/main/java/fr/neamar/kiss/RootModePreference.java

```diff
@@ -0,0 +1,33 @@
+package fr.neamar.kiss;
```

Comment:
I think we should move this class (and ResetPreference too) in a custom `preference` package